docs/machine-learning/resources/tasks.md

```diff
@@ -0,0 +1,46 @@
+# Machine Learning Tasks
+After understanding what you are hoping to achieve with your data, you will need to pick the right machine learning task for your situation. Below, you can find an explanation of different machine learning tasks and some common use cases. 
+
+> [!NOTE]
+> ML.NET is currently in Preview. All machine learning tasks are not currently supported. To submit a request for a certain task, please open an issue on our [GitHub repository](https://github.com/dotnet/machinelearning/issues).
+
+## Binary Classification
+A supervised machine learning task that is used to predict which of two classes an instance of data belongs to. The input of a classification algorithm is a set of labeled examples. Each example is represented as a feature vector, and each label is an integer of value of 0 or 1. The output of a binary classification algorithm is a classifier, which can be used to predict the label of new unlabeled instances. Examples of binary classification scenarios include:
```

Comment:
I'm wondering whether it is worth to get into details like "each example is represented as a feature vector", as it doesn't really add value in differentiating between the different scenarios.

Reply:
Agreed maybe the term "feature vector" could be a little too much, but I think it might be helpful to understand that the label will come back as "0 or 1" in this case (which would be different for a different task).